comixed-frontend/src/app/user/adaptors/authentication.adaptor.ts

```diff
@@ -22,7 +22,7 @@ import { Roles } from 'app/models/ui/roles';
 import { AppState, User } from 'app/user';
 import {
   AuthCheckState,
-  AuthHideLogin,
+  AuthHideLogin, AuthLoginFailed,
```

Comment:
Could you run Prettier on this commit and refresh the PR?

Reply:
I've added the Prettier plugin in IntelliJ but I'm not sure if the configuration is correct because there are some modifications on code I didn't touch.

Reply:
Prettier should pickup the configuration in the package.json file. We just need to keep the code formatted so commits are solely functional changes.

Be sure to only run the plugin on Typescript code, though. It gets weird on HTML (I haven'tgotten  that configured yet).